### PR TITLE
Data serializer register

### PR DESCRIPTION
**Pull request by benghaziboy**
@mparent61 @MichaelButkovic 

not ready yet.

Just looking for feedback.

Left it in bulbs, cause daddy wanted it there & it's currently light enough.

### CODE REVIEW

File: bulbs/serializer_registry/registry.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+        from bulbs.serializer_registry import registry
+        print(registry.serializers)
+        [<class 'MySerializer'>]
+        """
+        def wrapped(cls):
+            name = kwargs.get('name', 'serializers')
+            registered = getattr(self.__class__, name, dict())
+            key = kwargs.get('key', cls.__name__)
+            registered[key] = cls
```
Comment (mparent61) [inline bulbs/serializer_registry/registry.py]: Should validate that key doesn't already exist

File: tests/serializer_registry/test_registry.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+from django.test import TestCase
+
+from bulbs.infographics import data_serializers
+from bulbs.infographics.enum import InfographicType
+from bulbs.serializer_registry import registry
+
+
+
+
+class RegistryTestCase(TestCase):
```
Comment (mparent61) [inline tests/serializer_registry/test_registry.py]: I'd add a test case that creates a new serializer (inside the test def) w/ decorator, independent of Infographics

**Comment by mparent61** [comment]
Looking good

File: bulbs/infographics/data_serializers.py [hunk 3]
```diff
@@ -17,28 +20,34 @@ class ComparisonKeySerializer(serializers.Serializer):
     initial = serializers.CharField()
 
 
+# Top-level infographic serializers.
+@registry.serializer(name='infographics', key=InfographicType.COMPARISON)
```
Comment (mparent61) [inline bulbs/infographics/data_serializers.py]: Maybe instead of generic `name`, should be `group`/`category` or similar?

Comment (benghaziboy) [inline bulbs/infographics/data_serializers.py]: `path`?

Comment (mparent61) [inline bulbs/infographics/data_serializers.py]: path feels too file-system-y. 
app? 
module?

Comment (benghaziboy) [inline bulbs/infographics/data_serializers.py]: those feel like misnomers since they are both specific to django. I'll go with category

**Comment by mparent61** [comment]
Replaced by https://github.com/theonion/django-bulbs/pull/285

**Pull request closed without merging**